server.js

```diff
@@ -1,5 +1,19 @@
-const app = require('./app')
+const app = require('./app');
+const mongoose = require("mongoose");
+const { DB_HOST = "mongodb+srv://bogdan:qazwsxedc@cluster0.weuixzx.mongodb.net/db-contacts", PORT = 3000 } = process.env;
```

Comment:
mongo url is sensitive information. store it in .env only. you shouldn define constants from .env, only inport them here